### PR TITLE
Assess v3 docs migration

### PR DESCRIPTION
**Pull request by pzaichkina**
# Description

Migrating new Assess v3 categories docs

## Type of change

- [X] New document(s)/updating existing
- [ ] Fixes
- [ ] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: docs/assess/reports/enhanced-financials-ecommerce-lenders/balance-sheet.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+`GET /companies/{companyId}/reports/enhancedBalanceSheet/accounts`
+
+## Parameters
+
+In addition to the mandatory **companyId**, you can also include these parameters in your API call.
+
+| Parameter       	| Type            	| Description                                                                                 	| Required 	|
+|-----------------	|-----------------	|---------------------------------------------------------------------------------------------	|----------	|
+| **reportDate**     	| _string_<br>See Date</br> 	| YYYY-MM-DD.<br>Datetime or Date (inclusive of the whole day).</br>                                   	| Optional 	|
```
Comment (mcclowes) [inline docs/assess/reports/enhanced-financials-ecommerce-lenders/balance-sheet.md:25]:
```suggestion
| **reportDate**     	| _string_<br/>See Date 	| YYYY-MM-DD.<br/>Datetime or Date (inclusive of the whole day).                                   	| Optional 	|

```

**Review by mcclowes — DISMISSED** [review]
e.g.

**Comment by mcclowes** [comment]
Is this ready to merge?

**Comment by pzaichkina** [comment]
> Is this ready to merge?

nope, hence the draft status :) will be ready soon.

**Review by mcclowes — APPROVED** [review]
Trusting this is all correct 🤖👍

**Pull request merged**